requirements/test-requirements.txt

```diff
@@ -35,3 +35,5 @@ evaluate
 nltk
 rouge_score
 textstat
+# Required by auth tests
+psutil
```

Comment:
Without this, the`tests/server/auth/test_auth.py` tests failed in both dev environments:
* `dev/dev-env-setup.sh`
* `dev/run-test-container.sh`